google-cloud-dlp/Gemfile

```diff
@@ -6,13 +6,6 @@ gem "gcloud-jsondoc",
     git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
     branch: "gcloud-jsondoc"
 
-if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.1")
```

Comment:
I think we still need this conditional in the Gemfile in order to support Ruby 2.0 in the builds.